source/content/guides/decoupled-sites/04-site.md

```diff
@@ -219,3 +217,43 @@ You are directed to a new page with the site name, deployment, and build informa
 
 </Tab>
 </TabList>
+
+## Stable URLs
+
+You can establish FQDN domains in decoupled environments using stable URLs.
+
+An environment relates to every code change made against the Git repository that triggers a build, in which that build generates an internet-ready site.
+
+Code change events that trigger a build include:
+
+* Push to branches
+* Opened pull requests 
+
+<Alert title="Note" type="info">
+
+NOTE: Pull requests from a Multidev branch that are made against the upstream, will trigger double builds.
+</Alert>
+
+### Types of Environments
+
+
+* **Production environment** 
+    * The production environment is the default branch of the repository
+    * This environment corresponds to the following stable URL pattern: `live-[site-name].appa.pantheon.site`
+
+* **Multidev environment (based on integration branches)**
+    * Multidev environments are based on branches belonging to the upstream repository. This branch will have the prefix `multi-` in the branch name.
+    * This environment has the following stable URL pattern: `[branch-name]-[site-name].appa.pantheon.site`
+
+* **Multidev environment (based on pull requests)**
+    * This Multidev environment is built from all the opened pull requests against the upstream repository. 
```

Comment:
@joa-pan did some related testing today and I think it is more specific than this. Multidev environments based on pull requests appear to only be built for pull requests against the default branch, or integration branches (the ones prefixed with multi-) in the upstream repository.